### ISSUE TITLE
Dzienny limit wykorzystany?

### ISSUE THREAD

**Issue by lukzaj9** [allegro/allegro-api#8860]
Czy jest jakiś limit na dzienne dodawanie produktów przez API?
Otrzymaliśmy taki error:
"UserExceededSuggestionsLimit You have exceeded your daily submission limit. Please try again tomorrow."

**Comment by AureliuszBrussy** [comment]
Tak, taki limit istnieje. Więcej możesz przeczytać w [naszym newsie](https://developer.allegro.pl/news/wprowadzilismy-limit-produktow-ktore-sprzedajacy-moga-dodac-do-katalogu-w-ciagu-miesiaca-oraz-uruchomilismy-dodatkowa-walidacje-przy-wystawianiu-ofert-8dv6kygn2fk) (poprzedziliśmy to [zapowiedzią](https://developer.allegro.pl/news/wprowadzimy-limit-produktow-ktore-sprzedajacy-moga-dodac-do-katalogu-w-ciagu-miesiaca-oraz-uruchomimy-dodatkowa-walidacje-w-procesie-wystawiania-ofert-z-produktem-nnVXz3OodF8)).

**Comment by AureliuszBrussy** [comment]
@justynapalka to jest dzienny limit sugestii zmian w produktach. Więcej możesz przeczytać [w newsie](https://developer.allegro.pl/news/zgloszenia-bledow-w-produkcie-07-11-2023-wprowadzimy-dodatkowy-limit-zgloszen-w-ktorych-sprzedawca-moze-zglosic-zmiany-oraz-blokade-dodawania-wlasnych-wartosci-q05zEeZqRhL).
